src/OpenTelemetry.Exporter.Prometheus/PrometheusExporterOptions.cs

```diff
@@ -29,6 +29,7 @@ public class PrometheusExporterOptions
         internal Func<DateTimeOffset> GetUtcNowDateTimeOffset = () => DateTimeOffset.UtcNow;
 
         private int scrapeResponseCacheDurationMilliseconds = 10 * 1000;
+        private IReadOnlyCollection<string> httpListenerPrefixes = new string[] { "http://*:80/" };
```

Comment:
Now I start to wonder, do we want to have `80` as the default or `9090` (or depending on whether we are exposing it via `PrometheusExporterMiddleware` vs. `PrometheusExporterHttpServer`)?

Reply:
Doesn't have to be solved in this PR though.

Reply:
I'll investigate this further.

Reply:
I guess we can take 9090 to align with the default prometheus yaml file?

Reply:
9090 is the port of Prometheus application itself; so we should use something else as the port for the metrics we want Prometheus to collect data from. I'm curious about how did we come up with port 80 in the beginning. Thanks!

Reply:
#2780

Reply:
Updated the port to be 9184 as is in Noah's [example](https://github.com/dotnet/docs/blob/main/docs/core/diagnostics/metrics-collection.md#:~:text=opt.HttpListenerPrefixes%20%3D%20new%20string%5B%5D%20%7B%20%24%22http%3A//localhost%3A9184/%22%20%7D%3B ).